src/lang/english.txt

```diff
@@ -316,54 +318,54 @@ STR_SORT_BY_POPULATION                                          :Population
-STR_TOOLBAR_TOOLTIP_SWITCH_TOOLBAR                              :{BLACK}Switch toolbars
+STR_TOOLBAR_TOOLTIP_PAUSE_GAME                                  :{BLACK}Pause Game
+STR_TOOLBAR_TOOLTIP_FORWARD                                     :{BLACK}Fast Forward
+STR_TOOLBAR_TOOLTIP_OPTIONS                                     :{BLACK}Options & Settings
+STR_TOOLBAR_TOOLTIP_SAVE_GAME_ABANDON_GAME                      :{BLACK}Game Menu
+STR_TOOLBAR_TOOLTIP_DISPLAY_MAP                                 :{BLACK}Maps & Viewports
+STR_TOOLBAR_TOOLTIP_DISPLAY_TOWN_DIRECTORY                      :{BLACK}Town Directory
+STR_TOOLBAR_TOOLTIP_DISPLAY_SUBSIDIES                           :{BLACK}Subsidies
+STR_TOOLBAR_TOOLTIP_DISPLAY_LIST_OF_COMPANY_STATIONS            :{BLACK}Company Stations List
+STR_TOOLBAR_TOOLTIP_DISPLAY_COMPANY_FINANCES                    :{BLACK}Company Finances
+STR_TOOLBAR_TOOLTIP_DISPLAY_COMPANY_GENERAL                     :{BLACK}Company Information
+STR_TOOLBAR_TOOLTIP_DISPLAY_STORY_BOOK                          :{BLACK}Story Book
+STR_TOOLBAR_TOOLTIP_DISPLAY_GOALS_LIST                          :{BLACK}Scenario Goals
+STR_TOOLBAR_TOOLTIP_DISPLAY_GRAPHS                              :{BLACK}Graphs
+STR_TOOLBAR_TOOLTIP_DISPLAY_COMPANY_LEAGUE                      :{BLACK}Company Performance & Ratings
+STR_TOOLBAR_TOOLTIP_FUND_CONSTRUCTION_OF_NEW                    :{BLACK}Industries & Funding
+STR_TOOLBAR_TOOLTIP_DISPLAY_LIST_OF_COMPANY_TRAINS              :{BLACK}Company Train List & Groups{NBSP}-{NBSP}Ctrl+Click opens vehicle list only
+STR_TOOLBAR_TOOLTIP_DISPLAY_LIST_OF_COMPANY_ROAD_VEHICLES       :{BLACK}Company Road Vehicle List & Groups{NBSP}-{NBSP}Ctrl+Click opens vehicle list only
+STR_TOOLBAR_TOOLTIP_DISPLAY_LIST_OF_COMPANY_SHIPS               :{BLACK}Company Ship List & Groups{NBSP}-{NBSP}Ctrl+Click opens vehicle list only
+STR_TOOLBAR_TOOLTIP_DISPLAY_LIST_OF_COMPANY_AIRCRAFT            :{BLACK}Company Aircraft List & Groups{NBSP}-{NBSP}Ctrl+Click opens vehicle list only
+STR_TOOLBAR_TOOLTIP_ZOOM_THE_VIEW_IN                            :{BLACK}Zoom In
+STR_TOOLBAR_TOOLTIP_ZOOM_THE_VIEW_OUT                           :{BLACK}Zoom Out
+STR_TOOLBAR_TOOLTIP_BUILD_RAILROAD_TRACK                        :{BLACK}Build Railways
+STR_TOOLBAR_TOOLTIP_BUILD_ROADS                                 :{BLACK}Build Roads
+STR_TOOLBAR_TOOLTIP_BUILD_TRAMWAYS                              :{BLACK}Build Tramways
+STR_TOOLBAR_TOOLTIP_BUILD_SHIP_DOCKS                            :{BLACK}Build Waterways
+STR_TOOLBAR_TOOLTIP_BUILD_AIRPORTS                              :{BLACK}Build Airports
+STR_TOOLBAR_TOOLTIP_LANDSCAPING                                 :{BLACK}Landscaping Toolbar
+STR_TOOLBAR_TOOLTIP_SHOW_SOUND_MUSIC_WINDOW                     :{BLACK}Sound & Music
+STR_TOOLBAR_TOOLTIP_SHOW_LAST_MESSAGE_NEWS                      :{BLACK}News & Messages
+STR_TOOLBAR_TOOLTIP_LAND_BLOCK_INFORMATION                      :{BLACK}Help, Info & Screenshots
```

Comment:
Strictly speaking, there is no in-game help apart from the tooltips, so putting Help in the text is potentially misleading.

Reply:
Let's reserve it for the future for the time when we might actually have an in-game manual. It's what I want to do after I'm done fixing tooltips. :) All of the other things are what is usually found on the Help tabs of most software like text processors and stuff.

Reply:
This is resolved now!